src/schemathesis/schemas.py

```diff
@@ -44,8 +44,12 @@ def get_all_tests(
         for endpoint in self.get_all_endpoints():
             yield endpoint, create_test(endpoint, func, settings)
 
-    def parametrize(self, method: Optional[Filter] = None, endpoint: Optional[Filter] = None) -> Callable:
```

Comment:
Please, create a sentinel object, instead of a string and use it as a default everywhere for `method` or `endpoint`:

```
NOT_SET = object()
```